lib/final-api/endpoint/ddtf.rb

```diff
@@ -64,7 +64,7 @@ def registered(app)
 
             halt 400, enqueue_data.errors.to_json unless enqueue_data.valid?
 
-            config = DdtfHelpers.build_config(payload, tsd)
+            config = DdtfHelpers.build_config(payload, JSON.parse(enqueue_data.enqueue_data['TsdContent']))
```

Comment:
I don't like it either. I cannot use the #load_tsd method because the tsd is later modified by #normalize, #resolve_strategy, #build and #resolve_email. We could hide #load_tsd and the rest in one method (e.g. #build) and create an accesor for tsd. What do you think? @lksv @ondrej-hosak